aptos-move/aptos-vm/src/natives.rs

```diff
@@ -53,6 +54,14 @@ impl TAggregatorView for AptosBlankStorage {
     ) -> anyhow::Result<Option<StateValue>> {
         Ok(None)
     }
+
+    fn get_aggregator_v2_value(
+        &self,
+        _id: &Self::IdentifierV2,
+        _mode: AggregatorReadMode,
+    ) -> anyhow::Result<AggregatorValue> {
+        Err(anyhow::Error::msg(""))
```

Comment:
Since it's a blank storage, shouldn't we be returning some default value rather than an error message?

Reply:
I have to check things on end to end tests, but we shouldn't be asking for aggregators from storage, only from places that wrap MVHashMap or UnsyncMap

I can change these errors into unreachable!() and leave that for a next PR